### PR TITLE
ADD kichain snapshot script and instructions

### PR DESCRIPTION
**Pull request by MercurySolar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ http://cosmos-snap.staketab.com/ - Main page os snapshots.
 http://cosmos-snap.staketab.com/ixo - IXO (Mainnet)  
 http://cosmos-snap.staketab.com/medibloc - Medibloc (Mainnet)  
 http://cosmos-snap.staketab.com/stargaze - Stargaze (Mainnet)  
-http://cosmos-snap.staketab.com/sifchain - Sifchain (Betanet)  
+http://cosmos-snap.staketab.com/sifchain - Sifchain (Betanet)
```
Comment (icohigh) [inline README.md:14]: Need to change this link to - http://mercury-nodes.net/kichain-snaps/

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ http://cosmos-snap.staketab.com/ - Main page os snapshots.
...
```
Comment (duccas) [inline README.md:16]: Add please link to snap in Mainnet section (https://mercury-nodes.net/kichain-snaps/)

Comment (MercurySolar) [reply README.md:16]: Done

Comment (MercurySolar) [reply README.md:16]: Done

**Comment by MercurySolar** [comment]
Added https://mercury-nodes.net/kichain-snaps/ to the README.md

**Comment by duccas** [comment]
nice, merged

**Pull request merged**